youtube_dl/extractor/wdr.py

```diff
@@ -163,12 +151,35 @@ def _real_extract(self, url):
         else:
             ext = determine_ext(video_url)
 
-        description = self._html_search_meta('Description', webpage, 'description')
+        formats.append({'ext': ext, 'url': video_url})
+
+        m3u8_url = re.search(r'<li>\n<a rel="adaptiv" type="application/vnd\.apple\.mpegURL" href="(?P<link>.+?)"', webpage)
+
+        if m3u8_url is not None:
+            m3u8_url = m3u8_url.group('link')
+            formats.append({'ext': 'm3u8', 'url': m3u8_url})
+
+        webL_quality = -1
+        for video_vars in re.findall(r'<li>\n<a rel="(?P<format_id>web.?)"  href=".+?/(?P<link>fsk.+?)"', webpage):
```

Comment:
Using the whole `href` returns an error page (at least for me) that tells you to install flashplayer.